src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: $width-main;
+  padding: 50px 40px;
+  box-sizing: initial;
```

Comment:
why u need this?

Reply:
because without this my cards won't be positioned correctly. but I will try to change it because it isn't quite right.